src/less/overwrites/__overwrites.less

```diff
@@ -36,6 +36,15 @@ h6{
    }
 }
 
+h1{
+    font-weight: 400 !important;
```

Comment:
Please take a look at https://github.com/twbs/bootstrap/blob/master/less/type.less we should also add the .h1 there. Also i doubt we need the important? ;)